package.json

```diff
@@ -30,7 +30,8 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "postinstall": "node clean_packages.js"
```

Comment:
What do you think of moving the clean_packages.js to script/postinstall.js ? It would not add noise in the root folder and the name is IMO more explanatory ?